pages/Gold-Dust.md

```diff
@@ -0,0 +1,12 @@
+Gold Dust is one of the nine [dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts) added in Slimefun.
+
+## Obtaining
+Crushing Gold Ore in an [Ore Crusher](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Ore-Crusher) or an [Electric Ore Grinder](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ore-Grinder) yields two pieces of Gold Dust.
+
```

Comment:
You could maybe also mention that you can obtain it through "Gold Ore chunks" too, if you are using SlimefunOreChunks